Jenkinsfile

```diff
@@ -43,6 +44,12 @@ pipeline {
             description: 'This parameter declares whether to build ' +
                 'triton-origin-x86_64-21.4.0'
         )
+        booleanParam(
```

Comment:
Dumb DUMB question --> we in SmartOS-land really need to start worrying when it's 3-years-later and 24.4.x drops, right?

Reply:
I think maybe needing gcc13 might a more significant driver for updating the SmartOS build image.

This is primarily for building HVM images.